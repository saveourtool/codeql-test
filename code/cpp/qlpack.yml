name: codeql-code-test
dependencies:
  codeql-queries-test: "*"
extractor: cpp
tests: .