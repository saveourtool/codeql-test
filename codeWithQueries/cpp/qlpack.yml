name: codeql-code-with-queries-test
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .