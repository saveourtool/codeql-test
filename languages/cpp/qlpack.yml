name: codeql-test-examples
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .