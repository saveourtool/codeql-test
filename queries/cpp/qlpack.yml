name: codeql-queries-test
dependencies:
  codeql/cpp-all: "*"
extractor: cpp
tests: .