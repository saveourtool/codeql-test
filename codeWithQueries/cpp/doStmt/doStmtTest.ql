/**
 * @name Test 'do' statement
 * @kind problem
 * @id cpp/doStmtTest
 */

import cpp

from DoStmt doStmt
select doStmt, "This is 'do' statement."