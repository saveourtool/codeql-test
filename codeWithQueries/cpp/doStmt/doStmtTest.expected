| doStmtCode.cpp:10:5:15:32 | do (...) ... | This is 'do' statement. |
| doStmtCode.cpp:11:9:13:39 | do (...) ... | This is 'do' statement. |
| doStmtCode.cpp:21:5:23:32 | do (...) ... | This is 'do' statement. |
| doStmtCode.cpp:30:5:32:19 | do (...) ... | This is 'do' statement. |
| doStmtCode.cpp:38:5:40:41 | do (...) ... | This is 'do' statement. |
